src/components/domain/project/projectCard/ProjectSummaryCard.tsx

```diff
@@ -0,0 +1,156 @@
+import { cn } from '@/lib/utils';
+
+import TagInput from '@/components/common/input/TagInput';
+import ShadowCard from '@/components/common/card/ShadowCard';
+import ProjectVisibilityButton from '../projectButton/ProjectVisibilityButton';
+
+import { formatDateToDotSeparatedYYYYMMDD } from '@/lib/dateTime';
+import ProjectLinkButton from '../projectButton/ProjectLinkButton';
+import ProjectEvaluationButton from '../projectButton/ProjectEvaluationButton';
+import ProjectSendEvaluationLink from '../projectButton/ProjectSendEvaluationLink';
+import ProjectEditDeleteButton from '../projectButton/ProjectEditDeleteButton';
+
+/**
```

Comment:
개인적으로 상수를 분리하면 좋을 것 같네용
상수로 분리하면 좋은 점 : 
- `PROJECT_CARD_VARIANT.`를 입력하면 옵션이 자동으로 제안됨!
- 동일한 값을 참조할 때 항상 같은 상수를 사용
- 값을 변경해야 할 경우, 상수를 정의한 한 곳만 수정하면 됨
- 상수 이름을 통해 해당 값의 의미나 용도를 명확히 전달 가능 (ex. Admin vs PROJECT_CARD_VARIANT.ADMIN)
- 재사용 가능 (재사용하는 곳이 있을까요?)
- 타입 안전성을 높일 수 있다고 어느 곳에선가 본 것 같습니다..

```suggestion
/**const PROJECT_CARD_VARIANT = {
  ADMIN: 'Admin',
  LINK_PREVIEW: 'LinkPreview',
  MY_PROFILE: 'MyProfile',
  OTHER_PROFILE: 'OtherProfile',
  SEARCH_RESULT: 'SearchResult',
} as const;
```

보시고 어떤지 의견 부탁드립니다!

Reply:
좋읍니다! 그럼 variant 를 prop으로 넘길때도 좋겠네요 재사용은 많이 될 것 같습니다. (요약 조회가 필요한 페이지들에서 import)
이건 projectModels에 옮기겠습니다

Reply:
✅ ca8bfc2bf14379736820608cc842023bb111ebf6

수정했습니다~